tests/k6/src/test-issue-162.ts

```diff
@@ -13,7 +13,7 @@ export let options: Options = {
 };
 
 export default () => {
-    const res = api.uploadFile(defaults.accounts.einstein, files['kb_50.jpg'], `kb_50-${utils.randomString()}.jpg`)
+    const res = api.uploadFile(defaults.accounts.einstein, files['kb_50.jpg'], `kb_50-${__VU}-${__ITER}.jpg`)
```

Comment:
This is "a good thing" but probably not essential in this PR :) ?

Reply:
yes, but i used it to verify that everything runs smooth and found that little improvement ;)